zeeguu/core/model/article.py

```diff
@@ -195,6 +196,23 @@ def article_info(self, with_content=False):
         :return:
         """
 
+        # We don't need to store this in the DB.
+        # I was trying to use the self.compute_fk_and_wordcount()
+        # but this wasn't working to set the field?
```

Comment:
1. Indeed, I am thinking the _exp_ is more in-line with never rating an article absolutely 0, which I think for our case is fine enough. It's a bit hard to study the impact, but especially for Searches we should allow the system to explore the entire spectrum, maybe even increasing the offset since I think when searching we should prioritize matching on the actual content rather than recency / difficulty.
2. Yes, I think in the case we would use a very expensive way of calculating difficulty it makes sense to store it in the DB to avoid re-calculations, but for FK I think we could do it on-demand. Maybe we could have an application cache rather than a column in the DB? What do you think?

Reply:
While I looked at this, I also found about pagination - and I have given a test to the scroll and add more documents. It seems really easy to include it, so I could give it a try.

Right now, I was experimenting with scroll bottom to add more articles, but I need to figure out some details.